shell/platform/linux/fl_view.cc

```diff
@@ -179,18 +182,39 @@ static void check_pointer_inside(FlView* view, GdkEvent* event) {
   if (!view->pointer_inside) {
     view->pointer_inside = TRUE;
 
-    gdouble x, y;
+    gdouble x, y, pressure;
     if (gdk_event_get_coords(event, &x, &y)) {
       gint scale_factor = gtk_widget_get_scale_factor(GTK_WIDGET(view));
-
-      fl_engine_send_mouse_pointer_event(
-          view->engine, kAdd,
+      FlutterPointerDeviceKind kind =
+          check_device_is_stylus(view, (GdkEvent*)event, &pressure);
+      fl_engine_send_pointer_event(
+          view->engine, kAdd, kind,
           gdk_event_get_time(event) * kMicrosecondsPerMillisecond,
-          x * scale_factor, y * scale_factor, 0, 0, view->button_state);
+          x * scale_factor, y * scale_factor, 0, 0, view->button_state,
+          pressure);
     }
   }
 }
 
+FlutterPointerDeviceKind check_device_is_stylus(FlView* view,
```

Comment:
Should be `Flview *self`. Function should be called fl_view_check_device_is_stylus as it's a method.

Reply:
Renamed.